src/components/ebay-tab/index.marko

```diff
@@ -1,93 +1,2 @@
-    ...processHtmlAttributes(input, ignoredAttributes)
-    class=[
-        input.fake ? "fake-tabs" : "tabs",
-        input.class
-    ]>
-    <if(input.fake)>
-        $ var selectedPanel = input.panels[0];
-        <ul class="fake-tabs__items">
-            <for|heading, i| of=input.headings>
-                $ var isSelected = state.index === i;
-                <li
-                    ...processHtmlAttributes(
-                        heading,
-                        itemAnchorIgnoredAttributes
-                    )
-                    class=[
-                        heading.class,
-                        "fake-tabs__item",
-                        isSelected && "fake-tabs__item--current"
-                    ]>
-                    <a aria-current=(isSelected && "page") href=heading.href>
-                        <${heading.renderBody}/>
-                    </a>
-                </li>
-            </for>
-        </ul>
-        <div class="fake-tabs__content">
-            <div
-                ...processHtmlAttributes(selectedPanel)
-                class=[selectedPanel.class, "fake-tabs__panel"]>
-                <div class="fake-tabs__cell">
-                    <div>
-                        <${selectedPanel.renderBody}/>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </if>
-    <else>
-        <div class="tabs__items" role="tablist" key="headings">
-            <for|heading, i| of=input.headings>
-                $ var isSelected = state.index === i;
-                <div
-                    ...processHtmlAttributes(
-                        heading,
-                        itemAnchorIgnoredAttributes
-                    )
-                    key=`tab-${i}`
-                    id:scoped=`tab-${i}`
-                    aria-controls:scoped=`tabpanel-${i}`
-                    role="tab"
-                    aria-selected=(isSelected && "true")
-                    class=[heading.class, "tabs__item"]
-                    onClick("handleHeadingClick", i)
-                    onKeydown("handleHeadingKeydown", i)>
-                    <span>
-                        <${heading.renderBody}/>
-                    </span>
-                </div>
-            </for>
-        </div>
-        <div class="tabs__content">
-            <for|panel, i| of=input.panels>
-                $ var isSelected = state.index === i;
-                <div
-                    ...processHtmlAttributes(panel)
-                    id:scoped=`tabpanel-${i}`
-                    aria-labelledby:scoped=`tab-${i}`
-                    role="tabpanel"
-                    class=[panel.class, "tabs__panel"]
-                    hidden=!isSelected>
-                    <div class="tabs__cell">
-                        <div>
-                            <${panel.renderBody}/>
-                        </div>
-                    </div>
-                </div>
-            </for>
-        </div>
-    </else>
+<div>
```

Comment:
Why is this just a `<div>`? (We should be able to just delete this file and remove the template from the tag def).

Reply:
I had issues when there was no index.marko file together with a transformer. I can try again.

Reply:
When I remove the empty div I get this error: `Error: Invalid tag file: ebayui-core/src/components/ebay-breadcrumb/marko-tag.json. Neither a renderer or a template was found for tag.`
We do the same for the tags repo. If you have a better suggestion I can change it but I will leave it in for now.

Reply:
I updated to what you recommended. Now it uses the global marko.json